packages/backend/src/registries/ModelRegistryRegistry.ts

```diff
@@ -0,0 +1,51 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { Publisher } from '../utils/Publisher';
+import { Disposable, type Webview } from '@podman-desktop/api';
+import { Messages } from '@shared/Messages';
+import type { ModelRegistry } from '../models/ModelRegistry';
+
+export class ModelRegistryRegistry extends Publisher<string[]> {
```

Comment:
Can we found a better name?

Reply:
> Can we found a better name?

ModelTransport
ModelHandler
ModelProtocol

Reply:
Slight variants of ModelHandler 
- ModelsHandler (plural on model)
- ModelHandlers (plural on handler)

but anyway, ModelHandler would do the job for now

Reply:
I tried to follow the naming pattern used for InferenceProvider(Registry) so my proposition is that the abstract class is ModelHandler and register ModelHandlerRegistry